website/static/bloop-schema.json

```diff
@@ -306,12 +306,13 @@
           "items": {
             "$id": "/properties/project/properties/sources/items",
             "type": "string",
-            "title": "An example of a source directory",
+            "title": "An example of a source file or directory",
```

Comment:
Should directory values end with slash?

Reply:
In the JSON files it's ok if they don't end with slash, distinguishing them from single source files is simple as collecting all the files that don't end with `*.scala` and `*.java`.